apps/api/src/schema/programs.ts

```diff
@@ -282,3 +289,67 @@ export const ugradRequirementsResponseSchema = z.object({
     .array(programRequirementSchema)
     .openapi({ description: "The requirements in this requirements block" }),
 });
+
+export const standingyearEnum = z.enum(["Freshman", "Sophomore", "Junior", "Senior"]);
+
+export const sampleProgramsYearSchema = z.object({
+  year: standingyearEnum.openapi({
+    description: "Class standing or year level",
+    example: "Freshman",
+  }),
+  fall: z.array(z.string()).openapi({
+    description: "Courses recommended for Fall term",
+    example: ["I&C SCI 31", "MATH 2A", "WRITING 40"],
+  }),
+  winter: z.array(z.string()).openapi({
+    description: "Courses recommended for Winter term",
+    example: ["I&C SCI 32", "MATH 2B", "WRITING 50"],
+  }),
+  spring: z.array(z.string()).openapi({
+    description: "Courses recommended for Spring term",
+    example: ["I&C SCI 33", "IN4MATX 43", "WRITING 60"],
+  }),
+});
+
+export const sampleProgramVariationSchema = z.object({
+  label: z.string().optional().openapi({
+    description:
+      "Label describing this variation (empty string for programs with single variation)",
```

Comment:
I don't think we should explain why labels may be missing because it doesn't give much benefit and we shouldn't make promises that we can't keep. Users can draw an inference about when labels tend to be missing on their own time.